PRESIDECMS-2035 add missing rules filters duplicate feature in 10 13

pixl8-brayden opened the pull request.

pixl8-brayden committed:
[twgit] Init feature 'feature-PRESIDECMS-2035_add_missing_rules_filters_duplicate_feature_in_10_13'.

pixl8-brayden committed:
PRESIDECMS-2035 add missing rules filter duplicate feature in 10.13

DominicWatson commented:
Nice one, merging into upcoming 10.14 release for internal testing before releasing in hotfix.

pixl8-brayden committed:
PRESIDECMS-2035
- enable clone permisison in rules engine condition object
- add missing object title if empty
- update clone custom run event action
